addons/arsenal/functions/fnc_sortStatement_rateOfFire.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+/*
+ * Author: Alganthe, SynixeBrett
+ * Statement for the rate of fire sort.
+ *
+ * Arguments:
+ * 0: item config path (CONFIG)
+ *
+ * Return Value:
+ * Number to sort by
+ *
+ * Public: No
+*/
+
+private _fireModes = getArray (_this >> "modes");
```

Comment:
Other sort functions have `params ["_config"]` instead of `_this` directly. Change this for consistency.